packages/teleport/src/Discover/Discover.tsx

```diff
@@ -167,18 +169,24 @@ function SideNavAgentConnect({ currentStep }) {
           <Text bold>Resource Connection</Text>
         </Flex>
         <Box ml={4} mt={4}>
-          {agentStepTitles.map((step, index) => {
+          {agentStepTitles.map((stepTitle, index) => {
             let className = '';
             if (currentStep > index) {
               className = 'checked';
             } else if (currentStep === index) {
               className = 'active';
             }
 
+            // All flows will have a finished step that
```

Comment:
Instead of having it be '', should we just add a "Finished" step title?

Reply:
we can discuss it next week, keeping it as is until then